Traducir.Web/src/App/Components/SuggestionsTable.tsx

```diff
@@ -105,4 +107,38 @@ export default class SuggestionsTable extends React.Component<SuggestionsTablePr
                 });
             });
     }
+
+    renderDeleteButton(sug: SOStringSuggestion): JSX.Element {
```

Comment:
make all the functions arrow functions (so that they retain `this` even if not bound).

the change is super straightforward, just do

```
renderDeleteButton = (sug: SOStringSuggestion): JSX.Element => {